### ISSUE TITLE
Implement a place for customer resources

### ISSUE THREAD

**Issue by AnthonyRollins** [cal-itp/crm-helpdesk#164]
## High-Level To-Dos

- [x] Setup meeting with the website team (Marissa/Segacy/Kegan/Anthony)
~~- [ ] Connect domain to Hubspot knowledge base~~
~~- [ ] Customize Knowledge base template website~~
~~- [ ] Determine articles for the website (i.e., what are contactless payments?)~~
~~- [ ] Update Cal-ITP on new feature/product for Customer Support~~
~~- [ ] Implementation of the knowledge base~~

- [x] Develop a customer resources content spreadsheet
- [ ] Determine missing resources that we want up on the revamped website

Designs here: https://www.figma.com/file/TfWk1iDHdlt7bSA3DjQuQv/Mobility-Marketplace?node-id=5299-23559&t=VpFJLQ27mR3kNplK-0

**Comment by AnthonyRollins** [comment]
Same #165, maybe we rethink these issues

**Comment by o-ram** [comment]
@AnthonyRollins I went ahead and made some changes to this entire ticket to reflect my thinking given the path this has gone down:

1. I think we should get rid of the knowledge base label, as I don't see us using it. I created a new one called customer-resource, which I envision using for all issues involving a customer success resource that needs updating, creating, etc. 
2. I turned this ticket to reflect more on the pre-work we did ahead of the CalSTA proposal. I.e. we met with the team, and we developed that resource spreadsheet.
3. I made a new task about determining what was missing - I think that could turn into its own ticket(s) overtime as we look at things from the CS perspective and see what is needed for our customers.

I'm thinking we soon close this ticket and create other ones as the need arises. Thoughts? Any clear tasks in this realm in your mind right now that should be documented? I couldn't think of anything obvious, though I imagine with Marissa's news that we have preliminary approval to move forward with the site redesign, things will come up soon enough.
